core/sys/windows/kernel32.odin

```diff
@@ -3,75 +3,53 @@ package sys_windows
 
 foreign import kernel32 "system:Kernel32.lib"
 
-FOREGROUND_BLUE            :: WORD(0x0001)
-FOREGROUND_GREEN           :: WORD(0x0002)
-FOREGROUND_RED             :: WORD(0x0004)
-FOREGROUND_INTENSITY       :: WORD(0x0008)
-BACKGROUND_BLUE            :: WORD(0x0010)
-BACKGROUND_GREEN           :: WORD(0x0020)
-BACKGROUND_RED             :: WORD(0x0040)
-BACKGROUND_INTENSITY       :: WORD(0x0080)
-COMMON_LVB_LEADING_BYTE    :: WORD(0x0100)
-COMMON_LVB_TRAILING_BYTE   :: WORD(0x0200)
+FOREGROUND_BLUE :: WORD(0x0001)
```

Comment:
What happened to `kernel32.odin`? It's had its formatting screwed up on lines entirely unrelated to this PR.